src/scancode/cli.py

```diff
@@ -228,6 +234,7 @@ def validate_formats(ctx, param, value):
 @click.option('--diag', is_flag=True, default=False, help='Include additional diagnostic information such as error messages or result details.')
 @click.option('--timeout', is_flag=False, default=DEFAULT_TIMEOUT, type=int, show_default=True, help='Stop scanning a file if scanning takes longer than a timeout in seconds.')
 @click.option('--max-memory', is_flag=False, default=DEFAULT_MAX_MEMORY, type=int, show_default=True, help='Stop scanning a file if scanning requires more than a maximum amount of memory in megabytes.')
+@click.option('-I', '--ignore', is_flag=False, default='', callback=ignore_files, help='Do not scan <input> for these files')
```

Comment:
please no `-I` uppercase option. Just a single `--ignore` is enough.
Also we need an option that accepts only a single pattern not comma-separated and that can be repeated multiple times.